docs/maintain-guides-how-to-nominate-kusama.md

```diff
@@ -4,64 +4,32 @@ title: Be a Nominator (Kusama)
-_nominating_ to a reputable validator. This guide will walk you through how to do that during the soft launch phase.
-
-### Step 1: Bond your tokens
-
-During soft launch you will likely only have one account with KSM. This means that your _stash_ and _controller_ accounts
-will be the same account. After soft launch it is advised to set a new controller on your stash account, but for now
-this is fine. 
+## Step 1: Bond your tokens
 
 On the [Polkadot UI](https://polkadot.js.org/apps) navigate to the "Staking" tab.
 
-On this tab click on the "Account Actions" button on the top-left of the screen.
+The "Staking Overview" subsection will show you all the active validators and their information - points (reliability), earnings, identities, etc. The "Waiting" subsection lists all pending validators that need more nominations to enter the active validator set. The "Returns" screen will help you estimate your earnings and this is where it's good to start picking favorites. The "Account Actions" subsection allows you to stake and nominate and the "Validator Stats" screen will show you some interesting in-depth graphs about a selected validator.
 
-Click the blue "New Stake" button.
+Pick "Account Actions", then click the blue "New Stake" button.
 
 You will see a modal window that looks like the below:
 
-![Bonding](assets/guides/how-to-nominate/polkadot-dashboard-bonding.jpg)
-
-As mentioned before, **choose the same account for both controller and stash**.
-
-Select a "value bonded" that is **less** than the total amount of KSM you have, so you have some left over to transfer
-later when transfers are enabled. Try to save at least 500 mKSM to cover transferring to a new account and some fees.
+![Bonding](/img/NPoS/nominate2.png)
 
-Choose whatever payment destination sounds good to you. If you're unsure just click "Stash account (increase amount at
-stake)".
+Select a "value bonded" that is **less** than the total amount of KSM you have, so you have some left over to pay transaction fees. Be mindful of the reaping threshold - the amount that must remain in an account lest it be burned. That amount is 0.01 in Kusama, so it's recommended to keep at least 0.1 KSM in your account to be on the safe side.
 
-### Step 2: Nominate a validator 
+Choose whatever payment destination sounds good to you. If you're unsure choose "Stash account (increase amount at stake)".
 
-You are now bonded and can nominate up to 16 validators. Be careful about the validators you choose since you will be
-slashed if your validator commits an offence. 
+### Step 2: Nominate a validator
 
-Go to the "Staking Overview" tab.
+You are now bonded. Being bonded means your tokens are unlocked for slashing if the validators you nominate misbehave. Any bonded funds can now be distributed to up to 16 validators. Be careful about the validators you choose since you will be slashed if your validator commits an offence.
```

Comment:
```suggestion
You are now bonded. Being bonded means your tokens are locked and  could be slashed if the validators you nominate misbehave. All bonded funds can now be distributed to up to 16 validators. Be careful about the validators you choose since you will be slashed if your validator commits an offence.
```

Reply:
Can we add a link to the slashing page here for those who want to learn more about the offences?

Reply:
```suggestion
You are now bonded. Being bonded means your tokens are locked and could be [slashed](learn-staking#slashing) if the validators you nominate misbehave. Any bonded funds can now be distributed to up to 16 validators. Be careful about the validators you choose since you will be slashed if your validator commits an offence.
```